mysql-test/t/ps_trigger_access_check_regression.test

```diff
@@ -0,0 +1,52 @@
+#
+# PS-7778  Wrongly handled definer for triggers when prepared statements are used
+#
+
+--echo # Create users 'user01' and 'user02'.
+CREATE USER 'user01'@'%' IDENTIFIED BY 'user01';
+CREATE USER 'user02'@'%' IDENTIFIED BY 'user02';
+GRANT ALL PRIVILEGES ON *.* to 'user01'@'%';
+
+--echo #
+--echo # Fill tables.
+CREATE DATABASE test01;
+use test01;
```

Comment:
Multiple occurrences in the file:
s/use/USE

Reply:
I've rewritten the test, and hopefully have all keywords in upper case now.